en/docs/administer/configure-enterprise-login.md

```diff
@@ -97,3 +97,64 @@ Bring your own identity to Choreo by configuring a federated enterprise IdP on A
     ![Asgardeo applications](../assets/img/references/enterprise-login/sign-in-method.png){.cInlineImage-half}
 
 You are all set! Your users in the enterprise IdP can now log into the Choreo Console using their user credentials.
+
+
+## Role-based access control for Enterprise login
+
+Choreo allows you to configure your users residing in an external IDP (Identity Provider) to login into Choreo with appropriate permissions seamlessly based on their role.
+
+Follow the steps below to configure role-based access control to an enterprise login in Choreo:
+
+### Prerequisites
+
+1. Configure [Enterprise Login for your organization](https://wso2.com/choreo/docs/administration/configure-enterprise-login/)
+2. Make sure your Enterprise IDP includes the groups/roles attributes in the tokens it sends to Asgardeo in the respective protocol.
+3. Make sure you have admin privileges in Choreo.
+
+### Step1: Configure Asgardeo
+
+1. [Configure your IDP as an External IDP in Asgardeo](https://wso2.com/asgardeo/docs/guides/authentication/enterprise-login/). Depending on your IDP, you may select OpenID Connect or SAML as the protocol between Asgardeo and your IDP.
+
+    !!! note
+        If you are using OpenID Connect, configure the requested scopes accordingly for Asgardeo to get the relevant group/role details from the external IDP.
+
+        ![scopes](../assets/img/administration/configure-enterprise-login/configure-requested-scopes.png){.cInlineImage-half}
```

Comment:
This image is not loading

Reply:
This could be because it's inside nodes and the Github preview does not render notes.